src/components/indicatorMenu/IndicatorMenu.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+    <v-theme-provider theme="dark">
+        <v-btn v-if="appConfig && selectedIndicator" class="indicator-menu-activator">
+            <p class="text-wrap">
+                {{ appConfig.indicators[selectedIndicator].humanReadableName }}
+            </p>
+            <v-icon :icon="menuOpen ? 'mdi-chevron-down' : 'mdi-chevron-up'" end></v-icon>
+            <v-menu activator="parent" v-model="menuOpen">
+                <v-list class="bg-transparent opacity-80 elevation-0" style="max-width: min(35rem, 80%)">
+                    <v-list-item
+                        v-for="(id, index) in indicatorGroupIds"
+                        :key="id"
+                        class="bg-black mb-2 rounded pa-2 elevation-4"
+                        :active="id === selectedIndicator && !appConfig.indicatorGroups[index].subIndicators"
+                        :value="id"
+                        active-class="selected-item"
+                    >
+                        <router-link :to="`/${APP_BASE_ROUTE}/${id}/${selectedCountryId}`" custom v-slot="{ navigate }">
+                            <button class="text-left" type="button" @click="navigate">
+                                <ColourScaleIcon class="float-left mr-1" :size="24" :indicator="id"></ColourScaleIcon>
+                                <v-list-item-title>
+                                    {{ appConfig.indicators[id].humanReadableName }}
+                                </v-list-item-title>
+                                <v-list-item-subtitle class="pt-1">
+                                    {{ appConfig.indicators[id].description }}
+                                </v-list-item-subtitle>
+                            </button>
+                        </router-link>
+                        <v-slide-group
+                            v-if="appConfig.indicatorGroups[index].subIndicators"
+                            class="cursor-default"
+                            center-active
+                            :model-value="selectedIndicator"
+                            @click="slideGroupClicked($event)"
+                        >
+                            <IndicatorSlideGroupItem
+                                :indicator-id="id"
+                                :indicator-name="
+                                    appConfig.indicators[appConfig.indicatorGroups[index].mainIndicator].shortName
+                                "
+                            ></IndicatorSlideGroupItem>
+                            <IndicatorSlideGroupItem
+                                v-for="subId in appConfig.indicatorGroups[index].subIndicators"
```

Comment:
I couldn't (immediately) work out how 'all ages' is making its way into this property 'subIndicators', given that 'all ages' is the main indicator. Where in the code is the main indicator added to the list of sub indicators?